src/utils/WebResource.js

```diff
@@ -0,0 +1,24 @@
+window.tacticsUtils = window.tacticsUtils || {};
+
+(function () {
+    'use strict';
+
+    tacticsUtils.getWebResourceJSON = function (url){ 
+        return new Promise(function(resolve, reject) {
+           fetch(url).then(function(response) {
+               if (response.ok)
+                {
+                    resolve(response.json());
```

Comment:
Once you return the fetch promise, this "resolve" call can be a simple "return" statement.